### PR TITLE
Add support for Project badges

### PR DESCRIPTION
**Pull request by Miouge1**
Add support for project badges https://docs.gitlab.com/ee/api/project_badges.html

### CODE REVIEW

File: gitlab/v4/objects.py [hunk 1]
```diff
@@ -1250,6 +1250,17 @@ def enable(self, key_id, **kwargs):
...
 
+class ProjectBadge(SaveMixin, ObjectDeleteMixin, RESTObject):
+    pass
+
+
+class ProjectBadgeManager(CRUDMixin, RESTManager):
+    _path = '/projects/%(project_id)s/badges'
+    _obj_cls = ProjectBadge
+    _from_parent_attrs = {'project_id': 'id'}
+    _create_attrs = (('link_url', 'image_url'), tuple())
```
Comment (gpocentek) [inline gitlab/v4/objects.py]: It is also possible to update the badge objects, so could you define the `_update_attrs` variable as well?

File: gitlab/v4/objects.py [hunk 2]
```diff
@@ -2505,6 +2516,7 @@ class Project(SaveMixin, ObjectDeleteMixin, RESTObject):
         ('triggers', 'ProjectTriggerManager'),
         ('variables', 'ProjectVariableManager'),
         ('wikis', 'ProjectWikiManager'),
+        ('badges', 'ProjectBadgeManager'),
```
Comment (gpocentek) [inline gitlab/v4/objects.py]: Could you move this line to keep the list sorted?

**Comment by Miouge1** [comment]
@gpocentek thanks for the feedback. Can you review the latest commit, it should add tests, the update_attrs etc... ?

**Comment by gpocentek** [comment]
Thanks!

**Pull request merged**